### PR TITLE
OKTA-694170 OAUTH2-use-external-key-for-org-as-access-token

### PR DESCRIPTION
**Pull request by vanngo-okta**
## Description:
- **What's changed?** OAUTH2_USE_EXTERNAL_KEY_FOR_ORG_AS_ACCESS_TOKEN updates in AS concept, and Validate token guide.
- **Is this PR related to a Monolith release?** Yes, 2024.03.0

### Resolves:

* [OKTA-694170](https://oktainc.atlassian.net/browse/OKTA-694170)

### CODE REVIEW

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:70** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/concepts/auth-servers/index.md](https://github.com/okta/okta-developer-docs/blob/737a146367a3d60051c0012ea18ef2902976de57/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fconcepts%2Fauth-servers%2Findex.md) | [80](https://okta.acrolinx.cloud/api/v1/checking/scorecards/b6e06352-bcca-4ba7-8cba-a724e9cafd8f) | 64  | 93  | 42  | 96  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/validate-access-tokens/main/index.md](https://github.com/okta/okta-developer-docs/blob/737a146367a3d60051c0012ea18ef2902976de57/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fvalidate-access-tokens%2Fmain%2Findex.md) | [82](https://okta.acrolinx.cloud/api/v1/checking/scorecards/0e09e3fe-9242-405f-a6e5-6bbe15577fee) | 100  | 90  | 61  | 97  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.85569342-f3f8-4e0a-90bb-d8a74477e293?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Review by dannavarro-okta — APPROVED** [review]

**Review by grahamsmith-okta — DISMISSED** [review]

File: packages/@okta/vuepress-site/docs/guides/validate-access-tokens/main/index.md [hunk 1]
```diff
@@ -19,51 +19,53 @@ Validate access tokens.
...
+- Verify the claims found inside the access token.
 
 ### Retrieve The JSON Web Keys
 
-The JSON Web Keys (JWK) need to be retrieved from your [Okta authorization server](/docs/guides/customize-authz-server/), though your application should have them cached. See [Best practices](/docs/reference/api/oidc/#best-practices). Specifically, your authorization server's Metadata endpoint contains the `jwks_uri`, which you can use to get the JWK.  
+The JSON Web Keys (JWK) need to be retrieved from your [Okta authorization server](/docs/guides/customize-authz-server/), though your application should have them cached. See [Best practices](/docs/reference/api/oidc/#best-practices). Specifically, your authorization server's Metadata endpoint contains the `jwks_uri`, which you can use to get the JWK.
 
 > For more information about retrieving this metadata, see [Retrieve Authorization Server Metadata](/docs/reference/api/oidc/#well-knownoauth-authorization-server).
 
 ### Decoding and Validating the Access Token
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/validate-access-tokens/main/index.md:53]:
```suggestion
### Decode and Validate the Access Token
```

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:70** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/concepts/auth-servers/index.md](https://github.com/okta/okta-developer-docs/blob/fcf017e8a5705363ce20c492bb1ed98375277c60/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fconcepts%2Fauth-servers%2Findex.md) | [80](https://okta.acrolinx.cloud/api/v1/checking/scorecards/21162deb-ce8a-4e9f-bf68-a65e931c371b) | 64  | 93  | 42  | 96  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/validate-access-tokens/main/index.md](https://github.com/okta/okta-developer-docs/blob/fcf017e8a5705363ce20c492bb1ed98375277c60/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fvalidate-access-tokens%2Fmain%2Findex.md) | [83](https://okta.acrolinx.cloud/api/v1/checking/scorecards/8c311d38-e973-4304-9a2f-8c03da3e4004) | 100  | 90  | 66  | 97  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.d5b2b7da-3379-447d-b858-8f0db88b3dbc?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Review by susanharper-okta — APPROVED** [review]
LGTM

**Pull request merged**